tests/jsmarionette/runner/marionette-js-runner/host/python/runner-service/runner_service/handlers/runner.py

```diff
@@ -132,16 +134,58 @@ def __init__(self, *args, **kwargs):
     def start_runner(self, target, options):
         port = options.get('port')
         profile = Profile(profile=options.get('profile'))
-
-        print('---- start runner ----');
-        print(port, profile)
+ 
+        print("---- preparing to start runner ----")
+        print("port =", port, " with profile =", profile)
         self.runner.device.setup_port_forwarding(local_port=port, remote_port=port)
 
+        print("stopping b2g on device before starting runner.")
+        self.runner.device.dm.shellCheckOutput(['stop', 'b2g'])
+
+        # Clean up data on device.
+        print("cleaning up data on device.")
+        self.runner.device.dm.removeDir('/cache/*')
```

Comment:
nit: Perhaps make an array of the things you want to remove and then loop through them to keep things clean?